rhobs/alerting/data_plane/prometheus.availability_metric_konflux_alerts.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-konflux-up-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: konflux_up_alerts
+    interval: 1m
+    rules:
+    - alert: KonfluxAlert
+      expr: count ((konflux_up offset 1h) unless (konflux_up))
+      for: 1m
+      labels:
+        severity: warning
+      annotations:
```

Comment:
We need to set the `alert_routing_key` annotation to make sure the right team is tagged on the alert.

We should also define `runbook_url` but that can be deferred to a future update. Please create a task in the backlog if you choose to defer.

Reply:
@amisstea I have added `alert_routing_key` now but the `runbook_url` I haven't added for other alerts as well. I'll create a task for that later.